ui/src/containers/SituationList.vue

```diff
@@ -141,8 +141,8 @@ const filterList = (list: TSituation[]) => {
 
 			<div class="container">
 				<div class="autocomplete">
-					<div>
-						Result: {{ state.situations.length }} of
+					<div v-if="state.situations && state.situations.length">
```

Comment:
Should just be able to do `v-if="state.situations?.length"` here, but this works too